configure

```diff
@@ -787,6 +787,9 @@ def configure_arm(o):
 
   o['variables']['arm_fpu'] = options.arm_fpu or arm_fpu
 
+  if flavor == 'win':
+    o['msvs_windows_sdk_version'] = 'v10.0'
```

Comment:
`configure_arm` is called from `configure_node` which is called from `configure_engine` (https://github.com/nodejs/node-chakracore/blob/xplat/configure#L1338). At the point `configure_engine` is called, `output['target_defaults']` hasn't been defined yet- it gets defined later at https://github.com/nodejs/node-chakracore/blob/xplat/configure#L1376. So it looks like the way to get it propagated from `configure_arm` is to set the property directly on o here? Or did I misunderstand something?